shanoir-ng-front/src/app/dataset-acquisitions/dataset-acquisition/dataset-acquisition.component.html

```diff
@@ -149,6 +149,20 @@ <h2 class="header command-zone">Create dataset acquisition</h2>
 							</ng-template>
 						</span>
 					</li>
+                    <li *ngIf="mode != 'create' && datasetAcquisition.origin == 'SOURCE'">
```

Comment:
here if your acquisition is both a source and a copy you will not have every link